### ISSUE TITLE
Login credentials are now failing after Coway software update today

### ISSUE THREAD

**Issue by prsmith777** [sarahhenkens/home-assistant-iocare#23]
Something changed with this new update. Can no longer connect.

**Comment by skynet01** [comment]
yep.. same here. Any workaround? How will I keep my house clean?  This is a tragedy

**Comment by ampersandru** [comment]
same here - seems like Coway has a whole new login process, I had to transfer my old account to the new Coway account... so convoluted and confusing.

**Comment by sjafferali** [comment]
https://github.com/RobertD502/home-assistant-iocare/issues/12 < There is a maintained fork of this package where this issue is also being discussed. It looks like an update was just released for it that might address this issue.

**Comment by RobertD502** [comment]
@prsmith777 @skynet01 @ampersandru 

This repo hasn't been maintained in a while. I've been maintaining a new Coway integration over at https://github.com/RobertD502/home-assistant-iocare for ~1.5 years now. I just released version 0.1.4 which uses the new authorization method Coway recently implemented / caused this breaking change.

You can either add the repo manually to HACS or search for the repo within HACS as it is part of the "default" HACS repos.

**Comment by skynet01** [comment]
Thank you so much for posting this!
